.wp-env.json

```diff
@@ -0,0 +1,9 @@
+{
+  "core": null,
```

Comment:
You can leave this property out and it will default to the latest stable WordPress version.

Reply:
Left this as is, no danger, and we may well tidy up in a future iteration.